gpii/node_modules/couchConnector/index.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+var fluid = require("infusion");
+
+fluid.module.register("couchConnector", __dirname, require);
```

Comment:
This needs to be called gpii-couch-connector when/if we ever get to a micromodule setup, may as well get it right now

Reply:
Done.